<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QMainWindow" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>地大校园导航</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="View" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>终点：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>西区大门</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教二楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教三楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学一食堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学三食堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>52栋</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>主楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>信息楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图书馆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>弘毅堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>体育馆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>医院</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>大活</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>起点：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>西区大门</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教二楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>教三楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学一食堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>学三食堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>52栋</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>主楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>信息楼</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>图书馆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>弘毅堂</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>体育馆</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>医院</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>大活</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>搜索路径</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>显示全部路线</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>显示建筑</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>详细信息：</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QGraphicsView</extends>
   <header location="global">view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
